src/App.js

```diff
@@ -1,26 +1,72 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Grid from '@material-ui/core/Grid';
+import Container from '@material-ui/core/Container';
+import UserCard from "./components/userCard";
+import PostCard from "./components/postCard";
+import Button from '@material-ui/core/Button';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+    state = {
+        usersView: true,
+        users: [],
+        posts: []
+    };
+
+    constructor(props) {
+        super(props);
+        fetch(`https://jsonplaceholder.typicode.com/users`).then(
+            response => {
+                response.json().then(data => {
+                    this.setState( {users: data});
+                })
+            })
+    }
+
+    getPosts = async (userId) => {
+        const users = await fetch(`https://jsonplaceholder.typicode.com/posts?userId=${userId}`);
+        this.setState({posts: await users.json(), usersView: false});
```

Comment:
1. setState method should use callback as argument
2. What will happen if user request next page of posts?